### PR TITLE
Implemented: menu footer navigation component (#126)

### LINKED ISSUE #126: Implement menu footer navigation
## What is the motivation for adding/enhancing this feature?
Implement a menu footer navigation that will provide users with quick access to switch between Shopify config and product store.

- [ ] Pre-order
 
## What are the acceptance criteria?

- [ ] ...

## Can you complete this feature request by yourself?

- [ ] YES
- [ ] NO

### PR DESCRIPTION
**Pull request by sanskar345**
closes #126 
### Steps to verify:
- check that this component works well with apps (like job manager)
- proper working of ion-select
- Verify working of menu footer navigation in job manager app raised PR with changes https://github.com/hotwax/job-manager/pull/608
- Review changes of https://github.com/hotwax/job-manager/pull/608
### Additional Info

### CODE REVIEW

File: src/components/MenuFooterNavigation.vue [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      <ion-item v-else lines="none">
+        <ion-label class="ion-text-wrap">
+          {{ appUserState.currentEComStore.storeName }}
+        </ion-label>
+      </ion-item>
+      <!-- similarly, showing shopify configs only when there are multiple options to choose from 
+      but if both product store and config have multiple options, then only option to choose
+      product store will be visible -->
+      <ion-item v-if="appUserState.shopifyConfigs?.length > 1 && appUserState.userProfile?.stores?.length < 3" lines="none">
+        <ion-select interface="popover" :value="appUserState.currentShopifyConfig?.shopifyConfigId" @ionChange="$emit('changeShopifyConfig', $event)">
```
Comment (ymaheshwari1) [inline src/components/MenuFooterNavigation.vue:26]:
```suggestion
        <ion-select interface="popover" :value="appUserState.currentShopifyConfig?.shopifyConfigId" @ionChange="$emit('updateShopifyConfig', $event)">
```

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/components/MenuFooterNavigation.vue (hunk 1 above)
Comment (sanskar345) [reply src/components/MenuFooterNavigation.vue]: done

File: src/components/MenuFooterNavigation.vue [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        <ion-label class="ion-text-wrap">
+          <p class="overline">{{ instanceUrl }}</p>
+        </ion-label>
+        <ion-note slot="end">{{ appUserState.userProfile?.userTimeZone }}</ion-note>
+      </ion-item>
+      <!-- showing product stores only when there are multiple options to choose from. -->
+      <ion-item v-if="appUserState.userProfile?.stores?.length > 2" lines="none">
+        <!-- WHY EVENTS ($emit) IS USED WITH ION CHANGE: https://michaelnthiessen.com/pass-function-as-prop/ -->
+        <ion-select interface="popover" :value="appUserState.currentEComStore.productStoreId" @ionChange="$emit('updateEcomStore', $event)">
+          <ion-select-option v-for="store in (appUserState.userProfile?.stores ? appUserState.userProfile.stores : [])" :key="store.productStoreId" :value="store.productStoreId">{{ store.storeName }}</ion-select-option>
```
Comment (k2maan) [inline src/components/MenuFooterNavigation.vue:14]:
```suggestion
          <ion-select-option v-for="store in appUserState.userProfile.stores" :key="store.productStoreId" :value="store.productStoreId">{{ store.storeName }}</ion-select-option>
```

File: src/components/MenuFooterNavigation.vue [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+<script setup lang="ts">
+import { IonFooter, IonItem, IonLabel, IonNote, IonSelect, IonSelectOption, IonToolbar } from '@ionic/vue';
+import { appContext, useAuthStore } from "../index";
+import { computed } from 'vue';
+
+const authStore = useAuthStore();
+const appState = appContext.config.globalProperties.$store;
+const instanceUrl = computed(() => authStore.getOms); 
+const appUserState = computed(() => {
```
Comment (k2maan) [inline src/components/MenuFooterNavigation.vue:47]:
```suggestion
const userAppState = computed(() => {
```

**Review by k2maan — CHANGES_REQUESTED** [review]

**Review by k2maan — APPROVED** [review]

**Review by ymaheshwari1 — APPROVED** [review]

**Pull request merged**